apiserver/facades/client/modelmanager/modelmanager_test.go

```diff
@@ -78,17 +80,17 @@ type modelManagerSuite struct {
 
 var _ = gc.Suite(&modelManagerSuite{})
 
-func (s *modelManagerSuite) setUpMocks(c *gc.C) *gomock.Controller {
-	ctrl := gomock.NewController(c)
+func (s *modelManagerSuite) setUpMocks(c *gc.C) {
+	s.ctrl = gomock.NewController(c)
 
-	s.modelExporter = mocks.NewMockModelExporter(ctrl)
-	s.modelService = mocks.NewMockModelService(ctrl)
-	s.accessService = mocks.NewMockAccessService(ctrl)
-	return ctrl
+	s.modelExporter = mocks.NewMockModelExporter(s.ctrl)
+	s.modelService = mocks.NewMockModelService(s.ctrl)
+	s.accessService = mocks.NewMockAccessService(s.ctrl)
 }
 
 func (s *modelManagerSuite) SetUpTest(c *gc.C) {
 	s.IsolationSuite.SetUpTest(c)
+	s.setUpMocks(c)
```

Comment:
You can't do this. This is not the same `gc.C` and can cause issues with reporting. The fix is to place `setupMocks` for _every_ test.

Reply:
Ah, of course, yea. I actually mostly copied these test changes from a PR a while ago that I closed and fixed them up. I missed this.